src/cloud_agents/spawner/podman_spawner.py

```
@@ -64,13 +64,22 @@ async def _do_spawn(
         labels: dict[str, str] | None = None,
         skills_image: str | None = None,
         skills_paths: list[str] | None = None,
+        allowed_skills: list[str] | None = None,
         service_account: str | None = None,
         read_only: bool = False,
         credential_secret_name: str | None = None,
         mcp_secret_mounts: list[tuple[str, str, str]] | None = None,
         tls_certs: "EphemeralCerts | None" = None,
```

Ruff (0.16.2)

[error] Undefined name `EphemeralCerts`

(F821)